<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxC">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>121</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Celsius</string>
    </property>
    <widget class="QDial" name="dialC">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>50</width>
       <height>64</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumberC">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxF">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>121</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Fahrenheit</string>
    </property>
    <widget class="QDial" name="dialF">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>50</width>
       <height>64</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumberF">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
